.circleci/config.yml

```diff
@@ -26,7 +26,7 @@ jobs:
             env
             python3 -m venv .venv
             . .venv/bin/activate
-            pip install -U pip setuptools
+            pip install -U 'pip<19' setuptools
```

Comment:
Please remove this part of your patch.  I removed pyinstaller from requirements-ci.txt here: https://github.com/buildbot/buildbot/commit/093069f6ec727acbbb40da48de5b86e3e2846522#diff-3cd54b3c956e4935722ec3c8a8f09be6

Reply:
Wow, @tardyp is as fast as light 👍. Let me still answer it: pyinstaller is still used in CircleCI builds: https://github.com/buildbot/buildbot/blob/master/.circleci/config.yml#L32, and there's a broken build due to the referred pip issue: https://circleci.com/gh/buildbot/buildbot/7562. I've subscribed to the pip issue and will revert this change if the pip team fixes it and releases a new version.

BTW, apparently previous builds succeeded as pyinstaller was cached.